src/main/java/graphql/nadel/OperationType.java

```diff
@@ -0,0 +1,23 @@
+package graphql.nadel;
+
+public enum OperationType {
```

Comment:
Again the term "type" in graphql tends to mean the graphql types.

I would just call this "Operation" and have "name" and "displayName" as the attributes